src/esp/sim/Simulator.h

```diff
@@ -919,7 +920,7 @@ class Simulator {
    * @param assetInfo the asset to load
    * @param creation how to create the instance
    */
-  scene::SceneNode* loadAndCreateRenderAssetInstance(
+  gfx::replay::GfxReplayNode* loadAndCreateRenderAssetInstance(
```

Comment:
It feels weird to create this dependency on gfx-replay here in `Simulator.h`. I'm thinking about hypothetical other users of this function besides gfx-replay. It also seems more intuitive (and more useful) that `loadAndCreateRenderAssetInstance` would directly return a usable `SceneNode*`. We aren't trying to abstract away the concept of scene nodes in `Simulator.h`, right?

(In a similar vein, we can imagine that the new Magnum batch renderer's `addMeshHierarchy` will still return something like an `int` , not a `gfx::replay::GfxReplayNode*`.)

Reply:
This is reverted in ae16bbcca2925ec564a28ed7c9198a6028155844.